src/Libraries/PythonNodeModels/PythonNode.cs

```diff
@@ -83,6 +124,16 @@ private void SetEngineByDefault()
                 // Use CPython as default
                 engine = PythonEngineManager.CPython3EngineName;
             }
+            SetEnginePackageInfo(engine);
+        }
+        private void SetEnginePackageInfo(string engine)
+        {
+            var pkgDetails = PythonEngineManager.TryGetPythonEnginePackage(engine);
+            if (pkgDetails != null)
+            {
+                enginePackageName = pkgDetails.Item1;
+                enginePackageVersion = pkgDetails.Item2;
+            }
```

Comment:
I would expect that all engine names stored throughout Dynamo should be changed to something like FullyQualifiedEngineName (PackageName.EngineName@PkgVersion) So we should not need to match the EngineName to packages.